### PR TITLE
Remove .tekton and .konflux directories on release-next branches

### PR DESCRIPTION
**Pull request by creydr**
Remove .tekton and .konflux dirs on release-next branches (some prerequisite to disable Konflux on release-next branches)

### CODE REVIEW

**Comment by pierDipi** [comment]
LGTM, do you want to add the logic to delete the `.tekton` and  `.konflux` on main so that we don't have to do it manually ?

**Comment by creydr** [comment]
> LGTM, do you want to add the logic to delete the `.tekton` and `.konflux` on main so that we don't have to do it manually ?

Good point. Can you check on 2e493294106f122ec79c50689e25edb2cf18ebfe?

File: .github/workflows/release-generate-ci.yaml [hunk 1]
```diff
@@ -249,17 +216,6 @@ jobs:
...
-          GH_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
-          GITHUB_TOKEN: ${{ secrets.SERVERLESS_QE_ROBOT }}
-        if: ${{ (github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule') && github.ref_name == 'main' }}
-        name: '[serving - release-next] Create Konflux PR'
-        run: |
-          set -x
-          git remote add fork "https://github.com/serverless-qe/serving.git" || true # ignore: already exists errors
-          git push "https://serverless-qe:${GH_TOKEN}@github.com/serverless-qe/serving.git" sync-konflux/release-next:sync-konflux/release-next -f
-          gh pr create --base main --head serverless-qe:sync-konflux/release-next --title "[main] Add Konflux configurations" --body "Add Konflux components and pipelines" || true
-        working-directory: ./src/github.com/openshift-knative/hack/openshift-knative/serving
```
Comment (pierDipi) [inline .github/workflows/release-generate-ci.yaml:262]: without this PR, I don't the "removal" will get propagated, will them?

Comment (pierDipi) [reply .github/workflows/release-generate-ci.yaml:262]: what I'd do:
- keep konflux enabled
- remove `.tekton` and `.konflux` when branch `release-next`
- merge PR and run job
- merge created PRs
- remove special case logic and set konflux disabled

Comment (creydr) [reply .github/workflows/release-generate-ci.yaml]: > without this PR, I don't the "removal" will get propagated, will them?
uh. true.

Comment (creydr) [reply .github/workflows/release-generate-ci.yaml]: Updated

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**